test/integration_test/aws/run.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+# wait localstash pod status to running
+for ((k=0; k<30; k++)); do
+    not_run_num=`kubectl get pods -l app.kubernetes.io/name=localstack --no-headers | grep -v Running | wc -l`
+
+    if [ $not_run_num == 0 ]; then
+        break
+    fi
+
+    sleep 3
+done
+
+kubectl port-forward svc/localstack 4566:4566 &
```

Comment:
Do we need to cleanup this port-forwarding process?

Reply:
done in 69dcf65